SimplenoteWidgets/Views/NoteListRow.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+import WidgetKit
+
+struct NoteListRow: View {
+    var noteTitle: String
+    var width: CGFloat
+    var height: CGFloat
+
+    var body: some View {
+        Text(noteTitle)
+            .font(.subheadline)
+            .lineLimit(1)
+            .frame(width: width, height: height, alignment: .leading)
+        Divider()
+    }
+}
+
+struct NoteListRow_Previews: PreviewProvider {
+    static var previews: some View {
+        NoteListRow(noteTitle: "Title for note", width: 300, height: 50)
```

Comment:
Nipticky: Constants?

Reply:
That is just for the preview, it has no real affect on the actual compiled code...  In WPiOS, they removed all of the preview code.  I was thinking we should do the same thing.  
On one hand if we come to work on the UI later we have to add in these sorts of previews.  But on the other hand, how often will be be coming back to redesign the widget UI???  WDYT?